static/templates/compose.hbs

```diff
@@ -31,7 +31,12 @@
                 </button>
             </span>
             {{/unless}}
-            <span class="new_message_button">
+            <span class="new_message_button hide only-visible-for-spectators">
+                <a class="no-underline button small rounded float-left" href="/login" title="{{t 'Login in' }} (d)">
+                    {{t 'Login in to start conversation' }}
```

Comment:
Maybe "Log in to send messages" is a better button, and it should perhaps fill the whole width, since there's only one button action that makes sense?